front/app/containers/Admin/formBuilder/components/DeleteFormResultsNotice/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Link from 'utils/cl-router/Link';
+
+// styles
+import styled from 'styled-components';
+import { colors, fontSizes } from 'utils/styleUtils';
+
+// components
+import { Box, Icon, Text } from '@citizenlab/cl2-component-library';
+
+// intl
+import { InjectedIntlProps } from 'react-intl';
+import messages from '../messages';
+import { injectIntl } from 'utils/cl-intl';
+
+const StyledLink = styled(Link)`
+  color: ${colors.teal700};
+  font-size: ${fontSizes.m}px;
+  text-decoration: underline;
+  &:hover {
+    color: inherit;
+    text-decoration: underline;
+  }
+  white-space: nowrap;
+`;
+
+type Props = {
+  projectId: string;
+  redirectToSurveyPage?: boolean;
+} & InjectedIntlProps;
+
+const DeleteFormResultsNotice = ({
+  projectId,
+  redirectToSurveyPage,
+  intl: { formatMessage },
+}: Props) => (
+  <Box
+    width="100%"
+    display="flex"
+    flexDirection="row"
+    background={colors.teal100}
+    alignItems="center"
+    mb="16px"
+    px="16px"
+    data-cy="e2e-form-delete-results-notice"
+  >
+    <Icon name="alert-circle" fill={colors.teal700} />
+    <Text ml="16px" color="teal700" whiteSpace="nowrap">
+      {formatMessage(messages.disabledSurveyEditingMessage)}
+    </Text>
+    &nbsp;
+    {redirectToSurveyPage ? (
+      <StyledLink
```

Comment:
In what case do we not want to show the redirect link? I had some trouble finding it in the code.

Reply:
When we are on the Survey page. The one we redirect to. We are using the same notice there as-well with the exception that we don't redirect the user anywhere Url ending in `/native-survey`

Reply:
Ahh I see! Ok :) Thanks!